lib/bokken_web/router.ex

```diff
@@ -29,6 +37,21 @@ defmodule BokkenWeb.Router do
     get "/uploads/snippets/:user_id/:lecture_id/:file", FileController, :snippets
   end
 
+  scope "/bot_api", BokkenWeb do
+    pipe_through :api
+
+    scope "/auth" do
+      pipe_through :jwt_auth
+
+      post "/login", AuthBotController, :login
+    end
+
+    pipe_through [:jwt_auth, :ensure_auth]
+
+    get "/ninja/:discord_id", NinjaController, :show
+    patch "/ninja/:discord_id", NinjaController, :update
```

Comment:
```suggestion
    resources "/ninja", NinjaController, only: [:show, :update]
```

Reply:
You think it's better to have the bot having access to all of the Ninja endpoints? This way it would only allow to see and edit a Ninja, which is all that the bot need for now.